### DISCUSSION TITLE
Run GCAM maintaining historical conditions

### DISCUSSION THREAD

**Discussion by rubcalvo** [JGCRI/gcam-core#422] [Q&A]
Hi to all! thanks for this space to share our questions.

I'm trying to design some GCAM runs where available water changes according to some climate change scenarios. I want to compare the before and after of some water-related events. Hence it would be helpful to maintain the socioeconomic and technological conditions constant. Is there a way to do this? My main focus is water demand for the land module (agriculture).

I was thinking that maintaining the GPD, Population, and Agriculture Productivity Growth constant could be a good idea. Has anyone tried this before? Any idea?

Best regards,

**Comment by pkyle** [comment]
Water supply quantities are assigned in the `water_supply_constrained.xml` file, where each water basin's supply curve is defined as cost x quantity points. The `available` quantity indicates the portion of the `maxSubResource`, so the easiest way to increase or decrease the quantity available at any given price is to just change the `maxSubResource`, which is read in to each time period, but as a default the values are constant over time. The unit is km3/yr of renewable water supply.

**Comment by rubcalvo** [comment]
Thanks @pkyle! I was working with the `water_supply_constrained.xml` data. 

To clarify my question, I have an edited `water_supply_constrained.xml` with varying `maxSubResource` and want to compare the agricultural production before and after a significant change in water availability. Hence, I want to maintain sociotechnical conditions, e.g., in 2020, that significantly affect water and food demand. That's why I was thinking about GDP, Agricultural Productivity Growth, and Population. 
My question if anyone else has done something similar or thinks that other variable is relevant to consider.

Thanks!
